packages/echo-core/src/services/baseClient/baseClient.ts

```diff
@@ -34,7 +34,8 @@ export class BaseClient {
             );
             return authenticationResult ? authenticationResult.accessToken : '';
         } catch (exception) {
-            throw new AuthenticationError({ message: 'failed to authenticate', exception });
+            const typedException = exception as Record<string, unknown>;
```

Comment:
use toError(exception) instead

Reply:
Thanks for letting me know about this converter! Will update.